hoomd/mpcd/SphereGeometry.h

```diff
@@ -0,0 +1,197 @@
+
+#ifndef MPCD_SPHERE_GEOMETRY_H_
+#define MPCD_SPHERE_GEOMETRY_H_
+
+#include "BoundaryCondition.h"
+
+#include "hoomd/HOOMDMath.h"
+#include "hoomd/BoxDim.h"
+
+#include <cmath>
+
+#ifdef NVCC
+#define HOSTDEVICE __host__ __device__ inline
+#else
+#define HOSTDEVICE inline __attribute__((always_inline))
+#include <string>
+#endif // NVCC
+
+namespace mpcd
+{
+namespace detail
+{
+//! Spherical droplet geometry
```

Comment:
```suggestion
//! Sphere geometry
```
Please also fill in the documentation below. It doesn't have to be long, but it would be important to make it clear what this object represents. For example, you could say that this models a fluid confined inside a sphere centered at the origin and with radius R.